application/console/migrations/m160414_140140_email_queue.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use yii\db\Schema;
+use yii\db\Migration;
+
+class m160414_140140_email_queue extends Migration
+{
+    // Use safeUp/safeDown to run migration code within a transaction
+    public function safeUp()
+    {
+        $this->createTable(
+            '{{email_queue}}',
+            [
+                'id' => 'pk',
+                'to_address' => 'varchar(255) not null',
+                'cc_address' => 'varchar(255) null',
+                'subject' => 'varchar(255) not null',
+                'text_body' => 'text null',
+                'html_body' => 'text null',
+                'attempts_count' => 'tinyint not null default 0',
```

Comment:
I think the plan is to have a limit and check what the counter is before trying again, and once hit limit stop trying. Those checks and such still needed to be added though and we can talk through the logic of exponential back off on retries and what the limit should be.